cms/djangoapps/contentstore/views/course.py

```diff
@@ -1317,6 +1331,26 @@ def update_usage_info(store, course, configuration):
         configuration_json['usage'] = usage_information.get(configuration.id, [])
         return configuration_json
 
+    @staticmethod
+    def get_or_create_content_group_configuration(course):
```

Comment:
It's a shame that there are so many static methods here, but I can see that there's nowhere better to put them. It wouldn't make sense to put them all on the course because that class is already too heavy. It makes me wonder if there should be an intermediate object that could have these methods, e.g. something like a CourseGroupConfigurationsManger or a CourseGroupConfigurationsService.

Just thinking out loud... I wouldn't do anything about it now.